### DISCUSSION TITLE
New button for 2.1.45

### DISCUSSION THREAD

**Discussion by quangnd159** [realmayus/anki_forvo_dl#29] [Ideas]
Hi Marius! Thank you for this awesome addon! I know that 2.1.45 is still in beta as of writing, but it'd be great if you could implement a new button for the new refined button rows on the "Add" window. Something as simple as the word Forvo (no icon) would be great.

You must be very busy, so a No would be totally fine. I'd figured I'd just ask.

Regards,

Quang

**Comment by realmayus** [comment]
Hi, thanks for the comment. 
I haven't looked into that version yet, but I will turn this into a GitHub issue and will get back to it later 👍

**Comment by quangnd159** [comment]
Awesome! Thank you!

**Comment by realmayus** [comment]
This was fixed in yesterday's update :+1:
